generic/cells.cc

```diff
@@ -80,18 +80,16 @@ void lut_to_lc(const Context *ctx, CellInfo *lut, CellInfo *lc, bool no_dff)
     }
 
     if (no_dff) {
-        replace_port(lut, ctx->id("Q"), lc, ctx->id("Q"));
-        lc->params[ctx->id("FF_USED")] = 0;
```

Comment:
Might as well keep the FF_USED parameter, for devices where it matters

Reply:
Hmmm, true I guess.